In-house patch to add the location of PCRE header files for building the
C extension.  This patch is specific to Solaris and will not be
submitted upstream.

--- pyScss-1.3.4/setup.py.~1~	2015-10-06 02:08:05.983360539 -0800
+++ pyScss-1.3.4/setup.py	2015-10-06 02:09:00.989466001 -0800
@@ -33,7 +33,8 @@ speedups = Feature(
         Extension(
             'scss.grammar._scanner',
             sources=['scss/src/_speedups.c', 'scss/src/block_locator.c', 'scss/src/scanner.c', 'scss/src/hashtable.c'],
-            libraries=['pcre']
+            libraries=['pcre'],
+            include_dirs = ['/usr/include/pcre']
         ),
     ],
 )
